test/parallel/test-vm-is-context.js

```diff
@@ -26,14 +26,16 @@ const vm = require('vm');
 
 assert.throws(function() {
   vm.isContext('string is not supported');
-}, TypeError);
+}, /TypeError/);
 
 assert.strictEqual(vm.isContext({}), false);
 assert.strictEqual(vm.isContext([]), false);
 
 assert.strictEqual(vm.isContext(vm.createContext()), true);
 assert.strictEqual(vm.isContext(vm.createContext([])), true);
 
-const sandbox = { foo: 'bar' };
-vm.createContext(sandbox);
-assert.strictEqual(vm.isContext(sandbox), true);
+{
```

Comment:
I don't think it really matters either way, but I personally would probably opt to just wait until someone adds another test below it that uses variables.